README.md

```diff
@@ -25,8 +25,6 @@ To have more understanding of the tables contents. Please login to your Azure SQ
 
 ## Getting Started
 
-### Local Environment
```

Comment:
We should add this heading back when there are Codespaces/DevContainers headings, but for now, it doesnt need a heading.